src/Extensions/DibiPropertyClassReflectionExtension.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace staabm\PHPStanDba\Extensions;
+
+use Dibi\Row;
+use PHPStan\Reflection\ClassReflection;
+use PHPStan\Reflection\PropertiesClassReflectionExtension;
+use PHPStan\Reflection\PropertyReflection;
+use PHPStan\TrinaryLogic;
+use PHPStan\Type\IntegerType;
+use PHPStan\Type\Type;
+
+final class DibiPropertyClassReflectionExtension implements PropertiesClassReflectionExtension
+{
+    public function hasProperty(ClassReflection $classReflection, string $propertyName): bool
+    {
+        if (Row::class === $classReflection->getName()) {
+            return true;
+        }
+
+        return false;
+    }
+
+    public function getProperty(ClassReflection $classReflection, string $propertyName): PropertyReflection
+    {
+        // how to get the type of property from classReflection?
+
+        return new DibiRowPropertyReflection($classReflection, new IntegerType());
```

Comment:
we can't infer the type from the ClassReflection as the types are depedent on the GenericObjectType, not on the class level

Reply:
I tried using the stub files like suggest by Ondrej but couldnt make it work (while it works in the sandbox https://phpstan.org/r/4adcc760-1f94-45bb-a1ef-ac25c82a030b, it doesnt work in the PR)